Python/Product/Analysis/Values/FunctionInfo.cs

```diff
@@ -450,69 +450,75 @@ private class StringArrayComparer : IEqualityComparer<string[]> {
             }
         }
 
-        public override IEnumerable<OverloadResult> Overloads {
-            get {
-                if (_functionAttrs != null && _functionAttrs.TryGetValue("__wrapped__", out VariableDef wrapped)) {
-                    foreach (var o in wrapped.TypesNoCopy.SelectMany(n => n.Overloads)) {
-                        yield return o;
-                    }
+        public override IEnumerable<OverloadResult> Overloads => this.EnumerateOverloads(new HashSet<FunctionInfo>());
```

Comment:
Can we avoid the HashSet allocation in cases until we find a `__wrapped__` member?